Movie_MVVM/MovieMVVMUITests/MovieMVVMUITests.swift

```diff
@@ -0,0 +1,46 @@
+// MovieMVVMUITests.swift
+// Copyright © Avagyan Ernest. All rights reserved.
+
+import XCTest
+
+// UITests
```

Comment:
```suggestion
// UITests
```
Или все на русском, или все на английском